src/index.ts

```diff
@@ -479,7 +496,21 @@ export class UnleashClient extends TinyEmitter {
                         type: 'HttpError',
                         code: response.status,
                     });
+                    return;
                 }
+                if (response.status !== 304) {
+                    this.etag = response.headers.get('ETag') || '';
+                    const data = await response.json();
+                    await this.storeToggles(data.toggles);
+
+                    if (this.sdkState !== 'healthy') {
+                        this.sdkState = 'healthy';
+                    }
+
+                    this.emitReady();
+                }
+
+                this.updateLastRefresh();
```

Comment:
This should happen when we get a positive response from the server (2xx or 304).

Reply:
yes this is the case no?